gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -683,10 +681,10 @@ gst_tensor_filter_transform_caps (GstBaseTransform * trans,
         /* call setInputDimension with given input tensor */
         gst_tensors_info_init (&out_info);
         if (GST_TF_FW_V0 (priv->fw)) {
-          gst_tensor_filter_call (priv, res, setInputDimension, &config.info,
-              &out_info);
+          gst_tensor_filter_v0_call (priv, res, setInputDimension,
```

Comment:
This is not directly related to this PR. But I have some concerns on version checking.
If-statements to check version do not have the same control flow.

For example, some functions check both V0 and V1 using `GST_TF_FW_V0` and `GST_TF_FW_V1`.
But, another functions do this using either `GST_TF_FW_V0` or `GST_TF_FW_V1`, replying on `else` for the other version.

I think there is no problem currently. But, in near future, when newer versions are introduced, this makes some problems.

Reply:
There was the same discussion before with me and PK. At that time, this was kept because in this context v1/v2 check has been already done and assert-failed if it's not either v1 or v2. Anyway, as #2282 has suggested, we should not rely too much on assert (for NDEBUG) and we need to refactor these as well.

Reply:
Got it